src/hooks/useReport.ts

```diff
@@ -21,6 +14,18 @@ const useReport = (reportId?: string) => {
       const arrayBuffer = await response.arrayBuffer();
 
       const data = gunzipSync(new Uint8Array(arrayBuffer));
+      return data;
+}
+
+const useReport = (reportId?: string) => {
+  const queryResults = useQuery({
+    queryKey: ['report', reportId],
+    queryFn: async () => {
+      if (!reportId) {
+        throw new Error('reportId not available');
+      }
+
+      const data = await downloadReport(reportId);
```

Comment:
WDYT about adding this to the ario sdk? feels like maybe something for the gateway class we were discussing.

Reply:
I'm not aware of the gateway class discussions but seems reasonable add a utility function like this.